cmd/controller/main.go

```diff
@@ -433,6 +444,55 @@ type httpServer struct {
 	http.ServeMux
 }
 
+func runLeaderElection(ctx context.Context, cancel context.CancelFunc, logger *logrus.Entry, ctlConf config, kubeClient *kubernetes.Clientset, start func(_ context.Context)) {
+	if !ctlConf.LeaderElection {
+		start(ctx)
```

Comment:
nit: just return after this, rather than indenting the entire rest of the function

Reply:
done.